charts/victoria-metrics-k8s-stack/values.yaml

```
@@ -704,6 +704,15 @@ kubelet:
       insecureSkipVerify: true
       caFile: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
     bearerTokenFile: "/var/run/secrets/kubernetes.io/serviceaccount/token"
+    # drop high cardinality label and useless metrics for cadvisor and kubelet
+    metricRelabelings:
```

```suggestion
    metricRelabelConfigs:
```
metricRelabelings isn't supported by VMNodeScrape